melos.yaml

```diff
@@ -4,3 +4,8 @@ packages:
   - packages/noports_core
   - packages/sshnoports
   - packages/sshnp_gui
+
+command:
+  bootstrap:
+    hooks:
+      post: dart run melos exec --scope="noports_core" --scope="sshnoports" -- "dart run build_runner build --delete-conflicting-outputs"
```

Comment:
A post bootstrap hook which automatically builds version.dart in noports_core and sshnoports when you call melos bootstrap